src/components/services/LoanManagerService.ts

```diff
@@ -41,10 +44,12 @@ export const LoanManagerService = (props: Props) => {
   const dispatch = useDispatch()
   const clientId: string = useSelector(state => state.core.context.clientId)
   const loanAccountMap: LoanAccountMap = useSelector(state => state.loanManager.loanAccounts)
-
-  const state = useSelector(state => state)
+  const exchangeRates = useSelector(state => state.exchangeRates)
   const allTokens = useAllTokens(account)
 
+  const ltvEventTrigger = React.useMemo(() => Object.values(loanAccountMap), [loanAccountMap])
```

Comment:
What's the point of this `ltvEventTrigger`?

Reply:
Let's look into if we can remove this.

Reply:
No longer needed